### PR TITLE
FCT-1200: add `iconRight` to secondary- and primary-button component + 2 new icons

### PR DESCRIPTION
**Pull request by misama-ct**
## Description
The mentioned button components have been updated to include a new `iconRight` property and 2 new icons were added.

### Acceptance criteria:

* an iconRight property is added to the SecondaryButton  and PrimaryButton component
* styles and size of the icon are the same as in the existing iconLeft property.
* two new attached icons are added to UI KIT

### CODE REVIEW

File: packages/components/buttons/primary-button/src/primary-button.tsx [hunk 1]
```diff
@@ -127,6 +130,27 @@ const PrimaryButton = <TStringOrComponent extends ElementType = 'button'>(
   );
 
   const isActive = Boolean(props.isToggleButton && props.isToggled);
+
+  const renderIcon = (
```
[the remark this answers is not in the document]
Comment (misama-ct) [reply packages/components/buttons/primary-button/src/primary-button.tsx:134]: Hahaha, I didn't feel that grand today so I picked a "don't make me think"-ticket from our gap-filler lane and now you come around the corner nagging 😄

I'll fix it on monday, promised :)

[the remark this answers is not in the document]
Comment (tdeekens) [reply packages/components/buttons/primary-button/src/primary-button.tsx:134]: Haha, I know. I go by "unwanted nagger" with my middle name. Sorry about it. I do so when I feel lonely without coding ;)

**Review by tdeekens — APPROVED** [review]
🤗 many thanks!

**Review by ddouglasz — APPROVED** [review]
🙏🏾

**Pull request merged**